qiskit/aqua/utils/qp_solver.py

```diff
@@ -50,7 +50,7 @@ def optimize_svm(kernel_matrix: np.ndarray,
     Raises:
         NameError: CVXOPT not installed.
     """
-    # pylint: disable=invalid-name
+    # pylint: disable=invalid-name, unused-argument
```

Comment:
fwiw, depending on the solver you can set a max number of iterations. But it's hard to generalize when you don't know the solver being used ahead of time (it depends on what's installed on the users system). https://www.cvxpy.org/tutorial/advanced/index.html#setting-solver-options especially when not all solvers support it and the parameter name differs between solvers (osqp is `max_iter` but the other solvers that support it use `max_iters`) which is why I didn't pass this through.